### ISSUE TITLE
CLI argument for exiting if connection is already established

### ISSUE THREAD

**Issue by majorz** [resin-io/resin-wifi-connect#234]
Currently the example start script contains a few preconditions a user may choose from for starting WiFi Connect: is there a default gateway, google ping, etc. Preferably such functionality can be added in WiFi Connect directly. It should be implemented on top of the NetworkManager's API. It needs to be robust enough to work on slow networks. A new command line argument needs to be introduced for it, so that it does not break the current flow of the application in a non-backward compatible manner.

In other words, WiFi Connect may check on start whether the target interface is connected to a network and exit if that is not the case. If a connection is currently activating, it should wait for the result of the activation. This would be relatively easy to implement.
